sdk/core/Azure.Core/src/RequestUriBuilder.cs

```diff
@@ -278,16 +278,7 @@ public override string ToString()
                 stringBuilder.Append(PathSeparator);
             }
 
-            // TODO: Escaping can be done in-place
```

Comment:
What is the impact of this change?

Reply:
Two years ago, this code look like this: 
```cs
if (!HasQuery)
{
    stringBuilder.Append(_pathAndQuery);
}
else
{
    stringBuilder.Append(_pathAndQuery.ToString(0, _queryIndex));
    if (allowedQueryParameters == null)
    {
        stringBuilder.Append(_pathAndQuery.ToString(_queryIndex, _pathAndQuery.Length - _queryIndex));
    }
    else
    {
        AppendRedactedQuery(stringBuilder, allowedQueryParameters, redactedValue);
    }
}
```

It was changed in this PR: https://github.com/Azure/azure-sdk-for-net/pull/9782/files#diff-890ad460c6bc0ff005f2f636abc7fc7ddde1ef191a2f5407c4193e2ca31fcb28L284-L294